### PR TITLE
feat(front): invite users to project developer account[LET-546]

### PR DESCRIPTION
**Pull request by anamontiaga**
This PR contains the UI of Invitation Users to a project developer account feature.
The implemented solution has been to wrap an input with a div that resembles a textarea so that the chips can push the input.
_Table with users that has been added is in_ [PR 241](https://github.com/Vizzuality/heco-invest/pull/241)
We still don't have the endpoint so we can think about keeping task in progress by the moment.

## Testing instructions
- Go on `dashboard/users`
- Click Invite users
- Add some email and click Tab, Enter or a comma
- Remove one of them and check that text on input is removed 
- Click Send invite button and check emails logged on console are same as in textarea
- Close modal during the process and check is empty when you re-open.

Screenshot 2022-06-10 at 21 31 48

## Tracking
[LET-546](https://vizzuality.atlassian.net/browse/LET-546)

### CODE REVIEW

**Comment by anamontiaga** [comment]
@barbara-chaves I have added the error message when the text entered in the input does not correspond to an email (in `handleKeyDown` function) but it seems that something is missing in the code so that the errors are triggered.

File: frontend/containers/users/invite-users-modal/component.tsx
Comment (barbara-chaves) [inline frontend/containers/users/invite-users-modal/component.tsx:57]: We can remove it

**Review by barbara-chaves — APPROVED** [review]

**Comment by barbara-chaves** [comment]
I think we should wait until we have the correct endpoint to merge it into develop

**Pull request merged**
